reference/api/search.md

```diff
@@ -476,29 +485,58 @@ You will get the following response with the **cropped text in the `_formatted`
 
 **Parameter**: `cropLength`
 **Expected value**: a positive integer
-**Default value**: `200`
+**Default value**: `10`
 
-Configures the number of characters to keep on each side of the matching query term when using the [`attributesToCrop`](#attributes-to-crop) parameter. Note that this means there can be up to `2 * cropLength` characters in the cropped field.
+Configures the number of words to keep around the matching query term when using the [`attributesToCrop`](#attributes-to-crop) parameter. This value includes the query terms contained in the cropped text.
 
 If `attributesToCrop` is not configured, `cropLength` has no effect on the returned results.
 
+### Crop marker
+
+**Parameter**: `cropMarker`
+**Expected value**: a string
+**Default value**: `"…"`
```

Comment:
yeah, I noticed that. We should always use `…` since that's the actual ellipsis character. I was trying to keep the PR focussed on v0.27 stuff, but you're right, it's best to be consistent.

Reply:
Duh, just realised you meant the quotation marks. Yes, will fix that.